mobile/lib/features/wallet/send/send_lightning_screen.dart

```diff
@@ -253,7 +222,8 @@ class _SendLightningScreenState extends State<SendLightningScreen> {
                     ),
                     TenTenOneSwitch(
                         value: _payWithUsdp,
-                        showDisabled: false,
+                        isDisabled: usdpBalance <= 0,
```

Comment:
🤔 hmm.. isn't it nicer to let the user select the usdp option and show the user directly that they are missing funds. I feel like this could get annoying not knowing why the switch is disabled.

Reply:
https://github.com/get10101/10101/issues/1713